Initium-Odp/war/odp/javascript/script.js

```diff
@@ -4860,16 +4865,16 @@ $(document).keyup(function(event){
 	//	window.location.href='main.jsp';
 	//}
 	else if (event.which==85){ // U - go up in character list
-		switchCharacterByDirection(event, 1);
+		switchCharacter(event, null, 4);
 	}
 	else if (event.which==74){ // J - go down in character list
-		switchCharacterByDirection(event, 2);
+		switchCharacter(event, null, 4);
 	}
 	else if (event.which==75){ // K - iterate through characters in same acc and party
-		switchCharacterByDirection(event, 3);
+		switchCharacter(event, null, 4);
 	}
 	else if (event.which==76){ // L - switch to party leader if possible
-		switchCharacterByDirection(event, 4);
+		switchCharacter(event, null, 4);
 	}
```

Comment:
Fix direction parameter being sent to `switchCharacter` method.

Reply:
I also modified the `switchCharacter` method to accept a direction, and to sanitize the value if no direction is sent